dask_jobqueue/tests/test_jobqueue_core.py

```diff
@@ -275,25 +229,22 @@ def __init__(self, *args, **kwargs):
         MyCluster(cores=1, memory="1GB")
 
 
-@pytest.mark.parametrize("Cluster", all_clusters)
 def test_default_number_of_worker_processes(Cluster):
-    with create_cluster_func(Cluster, cores=4, memory="4GB") as cluster:
+    with Cluster(cores=4, memory="4GB") as cluster:
```

Comment:
Because I moved also the special handling of the `HTCondorCluster ` to the fixture even for local testing things have been becoming much more concise